spec/models/miq_provision_configuration_script_request_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe MiqProvisionConfigurationScriptRequest do
+  let(:admin)                { FactoryBot.create(:user) }
+  let(:ems)                  { FactoryBot.create(:ems_terraform_enterprise) }
```

Comment:
TODO can we validate the request task gets built without having to use TFE/AWX specifically